src/pages/search.tsx

```diff
@@ -450,7 +450,10 @@ const Search: NextPage<SearchProps> = ({
 
     return (
         <>
-            <NextSeo title={'Search | MongoDB'} />
+            <NextSeo
+                title={'Search | MongoDB'}
+                noindex={router.asPath === '/search/' ? false : true}
```

Comment:
Will this prevent it from indexing the search page itself?

Reply:
@mcolella14  No actually, `noindex` is only true if the search page has parameters. Otherwise, if the path value is `/search/` (landing page) then noindex would be false and page is indexed.